resources/qml/WelcomePages/WelcomeContent.qml

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2019 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+
+import UM 1.3 as UM
+import Cura 1.1 as Cura
+
+
+//
+// This component contains the content for the "Welcome" page of the welcome on-boarding process.
+//
+Column
+{
+    UM.I18nCatalog { id: catalog; name: "cura" }
+
+    spacing: 60
```

Comment:
Use the margins from the theme.

Reply:
Fixed in a9431b270. I've refactored this QML. It has a heavy trace of prototyping when I started out...